integration-tests/test_aqua.py

```diff
@@ -87,25 +93,28 @@ def test_install_spell_id(self):
         assert len(self.spell_id) != 0
 
     def test_install_get_config(self):
-        cfg_result = run_aqua(self.sk, "get_config", [self.spell_id])
+        cfg_result = run_aqua(self.key_pair_name, "get_config", [self.spell_id])
         assert cfg_result["success"]
-        assert cfg_result["config"] == self.config, "spell's config should be equal the one that was set during installtion"
+        assert cfg_result[
```

Comment:
Is it also autoformatting? Not a very intuitive place to add a newline.

Reply:
Fixed